cmd/main.go

```diff
@@ -36,6 +38,7 @@ func init() {
 	flag.BoolVar(&config.NoTeamsAPI, "noteamsapi", false, "Disable all access to the teams API")
 	flag.Parse()
 
+	config.IgnoredAnnotations = strings.Split(os.Getenv("IGNORED_ANNOTATIONS"), ",")
```

Comment:
Can be removed. Operator configuration should happen via ConfigMap or CRD. The other options here are for the operator deployment object.